orga/translations/en.json

```diff
@@ -1,15 +1,56 @@
 {
-  "current-lang": "en",
+  "api-errors": {
+    "global": "An error occurred. Please try again later."
+  },
+  "common": {
+    "actions": {
+      "cancel": "Cancel"
+    }
+  },
   "navigation": {
-    "main" : {
-      "campaigns" : "Campaigns",
+    "main": {
+      "campaigns": "Campaigns",
       "documentation": "Documentation",
-      "students-sco" :"Students",
-      "students-sup" :"Students",
-      "team" : "Team"
+      "students-sco": "Students",
+      "students-sup": "Students",
+      "team": "Team"
     },
     "user-logged-menu": {
       "logout": "Log out"
     }
+  },
+  "pages": {
+    "campaign-creation": {
+      "actions": {
+        "submit": "Create a campaign"
+      },
+      "external-id-label": {
+        "question-label": "Do you need an external user ID?",
+        "label": "external user ID label",
+        "suggestion": "Example : \"Student Number\" or \"Email address\" *"
+      },
+      "landing-page-text": {
+        "label": "Text to display on the starting page"
+      },
+      "legal-warning": "*In accordance with the French law governing computer technology and freedoms (“Informatique et Libertés”), and as data controller, please be careful not to ask for significant or identifying personal data unless it is absolutely necessary. Asking for social security numbers, as well as any sensitive data, is strictly prohibited.",
+      "name": {
+        "label": "Name of the campaign"
+      },
+      "no": "No",
+      "purpose": {
+        "assessment": "Assess participants",
+        "label": "What is the purpose of your campaign?",
+        "profiles-collection": "Collect the participants' Pix profiles"
+      },
+      "target-profile-informations": "If you want more information, see <a class=\"link\" href=\"https://cloud.pix.fr/s/3joGMGYWSpmHg5w\" target=\"_blank\" rel=\"noopener noreferrer\"> the corresponding documentation</a>.",
+      "target-profiles-list": {
+        "label": "What would you like to test?"
+      },
+      "test-title": {
+        "label": "Title of the personalised test"
+      },
+      "title": "Create a campaign",
+      "yes": "Yes"
+    }
```

Comment:
**suggestion (non-blockling) :**
Je pense qu'il faudrait séparer les traductions liées à la page et celles liées au modèle.
C'est un peu bizarre je trouve d'avoir label dans les clés de traductions.

```js
"pages": {
  "campaign-creation": {
    "title": "Create a campaign",
    "campaign": {
      "title": "Title of the personalised test"
    }
  }
}

```

Reply:
**suggestion (non-blocking) :** Je pense que dans l'idéale les traductions lié au modèle devrait être sortie de "pages" et être organisées sous une clé "models", mais  faut en discuter c'est peut-être pas une bonne idée et dans tout les cas c'est pas à faire dans ce ticket

Reply:
ici en fait c'est la traduction des champs du formulaire. Le découpage était fait comme suit :
```
champ: {
  label: ...,
  placeholder: ...,
}
```
Le champ en question c'est "Titre du parcours" traduit en anglais par "Title of the personalised test", c'est ce label qui est au dessus du champ, donc je trouve pas ça déconnant de découper ça comme ça.